lesson-10/public/app.js

```diff
@@ -0,0 +1,62 @@
+$('#button').click(() => {
+  const name = $('#myName').val();
```

Comment:
Try to make name and comment more distinguishable variable names just as it is in the selfie part